datasets/dataset_setup.py

```diff
@@ -436,19 +436,17 @@ def download_imagenet_v2(data_dir):
 
 
 def download_librispeech(dataset_dir, tmp_dir, train_tokenizer):
+  # pylint: disable=line-too-long
```

Comment:
nit: can you put this just on the lines that need it? you can put it as a trailing comment at the end of the line and it will affect just that line